packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -2,6 +2,32 @@
 
 The terra-functional-testing library is a utility for developing automation tests. The library extends the [WebdriverIO Framework](https://webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
 
+## Viewports
+
+Tests can be executed in any of the following supported viewports:
+- 'tiny' - width: 470; height 768
```

Comment:
We should be consistent with the semi colons:

```suggestion
- 'tiny' - width: 470; height: 768
```

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/402ee523e9eabedfc323de69d40318f8b075dd7d